### PR TITLE
Update website dataset docs

### PR DESCRIPTION
**Pull request by OleinikovasV**
Update dataset documention

### CODE REVIEW

File: docs/dataset.md [hunk 1]
```diff
@@ -15,46 +15,64 @@ sd_hide_title: true
...
+    |   |-- {two_char_code}.zip
+    |-- links # Apo and predicted structures similarity to their holo structures
+    |   |-- apo_links.parquet
+    |   |-- pred_links.parquet
+
+--------------------------------------------------------------------------------
+                            miscellaneous data below
+--------------------------------------------------------------------------------
+
+    |-- dbs # TSVs containing the raw files and IDs in the foldseek and mmseqs sub-databases
```
Comment (padix-key) [inline docs/dataset.md:41]:
```suggestion
    |   |-- pred_links.parquet
    |
--------------------------------------------------------------------------------
                            miscellaneous data below
--------------------------------------------------------------------------------
    |
    |-- dbs # TSVs containing the raw files and IDs in the foldseek and mmseqs sub-databases
```

File: docs/dataset.md [hunk 2]
```diff
@@ -63,47 +81,107 @@ Tables that lists all systems along with their annotations.
...
 
+:::{todo}
+- Add `Mandatory` column
+- Add `Example` column
+- Sort by mandatory columns
+- Show `system_id` as first column
+- Plugin [datatables](https://datatables.net/) to enable sorting, filtering and pagination
+- Then customize table style to fit all columns into the page
+:::
+
```
Comment (padix-key) [inline docs/dataset.md:92]: These TODOs have been addressed.

```suggestion

```

File: docs/dataset.md [hunk 3]
```diff
@@ -15,46 +15,64 @@ sd_hide_title: true
...
+    |       |-- pred.parquet
 ```
 
+:::{todo}
+- Add sections for missing tables
+- Fill missing elements into tables
+- Add descriptions for missing files
+:::
+
+We will describe the content of the `index`, `systems`, `clusters`, `splits`, `links` and `linked_structures` directories in detail below, the rest are described in the [miscellaneous section](#miscellaneous).
```
Comment (padix-key) [inline docs/dataset.md:74]: For some reason implicit targets do not work in *MyST*, but we can use explicit ones instead (see review comment below)

```suggestion
We will describe the content of the `index`, `systems`, `clusters`, `splits`, `links` and `linked_structures` directories in detail below, the rest are described in the [miscellaneous section](#miscellaneous-target).
```

File: docs/dataset.md [hunk 4]
```diff
@@ -116,111 +194,350 @@ This directory contains:
...
+*   - per_chain_bb_lddt_ave
+    - float
+    - Average per-chain lDDT score calculated over backbone atoms (N, CA, C, O)
 :::
 
-### Small molecule fingerprints (`fingerprints/`)
 
-Tables that contains all the ligand fingerprints used in calculating ligand similarity
-stored in `ligand_scores`.
+### Miscellaneous
```
Comment (padix-key) [inline docs/dataset.md:418]:
```suggestion
(miscellaneous-target)=
### Miscellaneous
```

File: column_descriptions/posebusters_checks.tsv [hunk 5]
```diff
@@ -1,126 +1,126 @@
 Name	Type	Description
```
Comment (padix-key) [inline column_descriptions/posebusters_checks.tsv:1]: As this is a TSV file, `tablegen.py` expects the columns to be separated by tab characters. This way also a new column is distinguished from whitespace between words in the descriptions.

In consequence, currently the descriptions of these annotations end up in the type column

File: docs/dataset.md [hunk 6]
```diff
@@ -15,46 +15,64 @@ sd_hide_title: true
...
-    |-- systems
-        |-- {two_char_code}.zip
+    |       |-- pred.parquet
 ```
 
+:::{todo}
+- Add sections for missing tables
+- Fill missing elements into tables
+- Add descriptions for missing files
+:::
```
Comment (padix-key) [inline docs/dataset.md:72]: These TODOs have been addressed.

```suggestion
```

File: docs/dataset.md [hunk 7]
```diff
@@ -63,47 +81,107 @@ Tables that lists all systems along with their annotations.
...
+- `protein_seqsim_max`: Global protein sequence similarity between components of two systems, aggregated by max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_max`:  Global  protein sequence similarity between components of two systems multiplied by query system coverage,  aggregated by max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_weighted_max`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_weighted_sum`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+- `protein_seqsim_weighted_max`:  Global protein sequence similarity between components of two systems, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
```
Comment (padix-key) [inline docs/dataset.md:168]: Just a nitpick, but I would prefer nested list where all values are formally stated, i.e.

```
- `cluster`: the cluster algorithm used
  - `communities`: community detection algorithm
  - `components`: ...
- `directed`: type of graph used for cluster input
  - `False`: undirected
  - `True`: directed
- `metric`: the similarity metric used for generating the clusters
  - `pli_qcov`: Protein-ligand interaction similarity between aligned ligand-binding region (pocket) residues of two systems.
  - ...
- `threshold`: similarity threshold in percent. 
  - `50`
  - `70`
  - ...
```

File: docs/dataset.md [hunk 8]
```diff
@@ -63,47 +81,107 @@ Tables that lists all systems along with their annotations.
...
 
 ### Splits (`splits/`)
 
-This directory contains:
+This directory contains split files and the configs used to generate them.
 
-- `plinder-pl50.parquet`: listing the split category for each system
-- ...
+- `split.parquet`: listing the split category for each system
+- `split.yaml`: the config used to generate the split
```
Comment (padix-key) [inline docs/dataset.md:178]: Do we need to describe the content of the yaml file as well?

File: docs/dataset.md [hunk 9]
```diff
@@ -116,111 +194,350 @@ This directory contains:
...
+    - Similarity metric of interest
+*   - source
+    - str
+    - Source of similarity metric. It could either be `foldseek`, `mmseqs` or `both`
+*   - metric
+    - str
+    - Similarity metric of interest
+*   - mapping
+    - str
+    - ??
```
Comment (padix-key) [inline docs/dataset.md:540]: Missing description

**Review by padix-key — COMMENTED** [review]
I left some small requests, but overall I think this is structured very nicely

File: docs/dataset.md [hunk 10]
```diff
@@ -15,46 +15,64 @@ sd_hide_title: true
...
 ```
 
+:::{todo}
+- Add sections for missing tables
+- Fill missing elements into tables
+- Add descriptions for missing files
+:::
+
+We will describe the content of the `index`, `systems`, `clusters`, `splits`, `links` and `linked_structures` directories in detail below, the rest are described in the [miscellaneous section](#miscellaneous).
+
```
Comment (padix-key) [inline docs/dataset.md:75]: The manifest could also be moved here.
Note that in the directory structure above `2024-04` and `v2` then also need to be replaced by the `release` and `iteration` placeholders below.

```suggestion
### Dataset versioning

The dataset is versioned using two identifiers:

- `release`: the time stamp of the last RCSB sync
  - `2024-04`
  - `2024-06`
- `iteration`: iterative development within a release
  - `v1`
  - `v2`
```

File: docs/dataset.md (hunk 8 above)
Comment (OleinikovasV) [reply docs/dataset.md:164]: I would say not really necessary for the user, more for developers.
What would be a good way to express this?

File: docs/dataset.md (hunk 10 above)
Comment (OleinikovasV) [reply docs/dataset.md:70]: yes, manifest.json might be better! will likely defer this to another branch then after a sync with @tjduigna

File: column_descriptions/posebusters_checks.tsv (hunk 5 above)
Comment (yusuf1759) [reply column_descriptions/posebusters_checks.tsv:1]: Makes sense.

File: docs/dataset.md (hunk 8 above)
Comment (padix-key) [reply docs/dataset.md:164]: As the file is quite short, I would propose to print an example file with a short comment describing the parameter, analogous to the directory listing:

```
sampling_cluster_metric: pli_qcov  # ...
sampling_cluster_threshold: 70     # ...
...
```

File: docs/dataset.md [hunk 11]
```diff
@@ -116,111 +184,351 @@ This directory contains:
...
-    - Cluster used in de-leaking the training vs test dataset
+    - Cluster label used in sampling test set
 *   - cluster_for_val_split
     - str
-    - Cluster used in sampling validation set from training set
-:::
+    - Cluster label used in sampling validation set from training set
+*   - uniqueness
+    - str
+    - system label used to remove redundant systems from the split
```
Comment (Ninjani) [inline docs/dataset.md:196]: redundant systems are only removed from val/test, they're kept in train

File: docs/dataset.md [hunk 12]
```diff
@@ -116,111 +184,351 @@ This directory contains:
...
-*   - similarity
+*   - id
+    - str
+    - The PDB or AF2DB (for `apo` and `pred`, respectively) `{source_id}_{chain_id}` tag
+*   - pocket_fident
+    - float
+    - sequence identity for pocket residues
+*   - pocket_lddt
+    - float
+    - Local Distance Difference Test (lDDT) score for the pocket residues
```
Comment (Ninjani) [inline docs/dataset.md:266]: Local Distance Difference Test (lDDT) score for the pocket residue alpha carbons as returned by Foldseek

File: docs/dataset.md [hunk 13]
```diff
@@ -116,111 +184,351 @@ This directory contains:
...
+    - Sum of fident for all templates, weighted by the number of residues in the template
+*   - protein_lddt_weighted_sum
+    - float
+    - Sum of lDDT for all residues, weighted by the number of residues in the template
+*   - target_id
+    - str
+    - apo or pred stucture `{source_id}` tag
+*   - sort_score
+    - float
+    - score used to sort linked structures
```
Comment (Ninjani) [inline docs/dataset.md:281]: this is resolution for apos and plddt for preds

File: docs/dataset.md [hunk 14]
```diff
@@ -67,43 +80,98 @@ Tables that lists all systems along with their annotations.
...
+  - `protein_seqsim_qcov_max`:  Global  protein sequence similarity between components of two systems multiplied by query system coverage,  aggregated by max score across all pairs of protein chains or ligand chains.
+  - `protein_seqsim_qcov_weighted_max`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+  - `protein_seqsim_qcov_weighted_sum`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+  - `protein_seqsim_weighted_max`:  Global protein sequence similarity between components of two systems, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+  - `protein_seqsim_weighted_sum`: Global protein sequence similarity between components of two systems, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
 - `threshold`: similarity threshold in percent.
```
Comment (OleinikovasV) [inline docs/dataset.md:154]: @yusuf1759 , some thresholds differ and go lower for some, but not all, i guess this is okay, or we can add "etc"

**Comment by tjduigna** [comment]
Note; whoever goes to merge this can add `bumpversion skip` to the commit message to avoid publishing a version of `plinder` to PyPI with no source code changes. It should still build the updated docs.

**Review by tjduigna — APPROVED** [review]
